docs/ref/forms/fields.txt

```diff
@@ -369,6 +369,13 @@ For each field, we describe the default widget used if you don't specify
     If provided, these arguments ensure that the string is at most or at least
     the given length.
 
+.. versionadded:: 1.8
```

Comment:
This should be roughly:
```
.. attribute:: strip

    .. versionadded:: 1.9

    Description
```
the text under max/min_length should be indented similarly I think